### PR TITLE
Text - remove default text alignment

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
⚠️ Text - remove default text alignment
I've checked with 'animated' prop and don't see any effect as the RN default is 'auto'

## Changelog
⚠️ Text - remove default text alignment

## Additional info
solves #2457

### CODE REVIEW

File: src/components/text/index.tsx [hunk 1]
```diff
@@ -157,7 +157,6 @@ class Text extends PureComponent<PropsTypes> {
 const styles = StyleSheet.create({
   container: {
     backgroundColor: 'transparent',
-    textAlign: 'left',
```
Comment (ethanshar) [inline src/components/text/index.tsx:160]: Did you check RTL? 
Also, did you check behavior on Android, it tends to be different sometimes..

**Review by ethanshar — APPROVED** [review]
Approved. 
But see my comment, also notice it will change private uilib snapshot, so be ready to fix these after it's merged

File: src/components/text/index.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply src/components/text/index.tsx:160]: Actually Android is acting fine with and without this line. Yep, problem in RTL - good catch!

**Comment by Inbal-Tish** [comment]
This solution breaks text alignment in RTL (only works on Android if the string language is actually on RTL language).

**Pull request closed without merging**